multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/UpdateServiceCredentialsStep.java

```diff
@@ -19,28 +18,18 @@
 @Scope(BeanDefinition.SCOPE_PROTOTYPE)
 public class UpdateServiceCredentialsStep extends ServiceStep {
```

Comment:
Should be renamed to `UpdateServiceParametersStep`.

Reply:
Done